src/pages/Home.js

```diff
@@ -111,6 +112,16 @@ class Home extends Component {
                             </ResultsScrollView>
                         )}
                 </SearchWrapper>
+                <div
+                    style={{
+                        padding: '24px',
+                        margin: '12px',
+                        backgroundColor: 'white',
+                    }}
+                >
+                    <RouterLink to="/start">Start Screen</RouterLink>
```

Comment:
I wouldn't worry too much about wiring up all these URLs, we're not going to use them for managing the game or anything

Reply:
Yeah, it's just a stopgap until we've got the FSM managing this rendering - just makes it slightly easier to get to me start page so you can look at it! I assume this Home page will be completely re-written by the time we get to MVP.